data/examples.py

```diff
@@ -0,0 +1,457 @@
+"""
```

Comment:
We shouldn't need this file - we should be doing all these code chunks in the jupyter book.

Reply:
But we don't want to actually simulate it, right? Because then it might change depending on e.g. the GSL version. So if it was created by msprime, and exact topology is important, we should use a saved `.ts` file, I assume?

Reply:
Yes, I think the pattern of having a hidden cell with code to generate the example ts and saving the ts in the data directory is the way to go. So we don't need this file here.